packages/cli/src/spec/configSpec.ts

```diff
@@ -80,7 +78,12 @@ describe('config', () => {
         [homedir()]: {},
       });
       const mockLog = new MockLog();
-      await loadOrCreateConfig(mockLog);
+      const mockPrompt = new MockPrompt();
+      // Since 'createConfig' will be called, we push 3 future answers to 'mockPrompt'.
+      mockPrompt.addMessage('false');
+      mockPrompt.addMessage('jdk');
+      mockPrompt.addMessage('sdk');
```

Comment:
Do you mind adding comments saying what these messages are meant to be answering? Just something like:

```
mockPrompt.addMessage('false'); // Should bubblewrap to install the JDK?
mockPrompt.addMessage('jdk'); // What is the path to the JDK?
```